pkg/resources/testdata/TestAcc_ApiAuthenticationIntegrationWithJwtBearer/basic/test.tf

```diff
@@ -0,0 +1,6 @@
+resource "snowflake_api_authentication_integration_with_jwt_bearer" "test" {
+  name                   = var.name
```

Comment:
lulz no enabled? I guess it's optional here :/

Reply:
This test is skipped for now, so I didn't notice this. It's required, so I added it.